### ISSUE TITLE
JSON Profile of GeoXACML

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ideas#95]
OASIS is developing a JSON Profile for XACML.

http://docs.oasis-open.org/xacml/xacml-json-http/v1.1/csprd01/xacml-json-http-v1.1-csprd01.html

It might be worth experimenting with it, once it is approved as an OASIS standard, with the idea being to develop a JSON Profile for GeoXACML.

**Comment by hector-rodriguez** [comment]
This JSON Profile opens up possibilities  within the realm JSON-based standards such as OAuth2.0 and User Managed Access (UMA) to generate PDP/PEP Authorization schemes.

Therefore, testing the generation of Policy Decision Points based on UMA, using this JSON XACML profile could be proven as a useful addition to the Federated Clouds Security field

**Comment by gbuehler** [comment]
On a related note, in T-15 we are touching “ALFA for Geo”, that is using the OASIS standard of ALFA to write policy statements in GeoXACML3.  Currently, GeoXACML 3 is just a discussion paper and probably needs to be advanced to deal with new requirements, including a JSON profile.
Axiomatics has a ALFA generation tool that we are using.  I think they would be keen to get involved with any future work.
